examples/energy_mng/Cargo.toml

```diff
@@ -17,15 +17,16 @@ name = "energy_mng"
 [dependencies]
 slint = { path = "../../api/rs/slint", default-features = false, features = ["compat-0-3-0"] }
 mcu-board-support = { path = "../mcu-board-support", optional = true }
+chrono = "0.4.23"
+weer_api = { version = "0.1", optional = true }
 
 [build-dependencies]
 slint-build = { path = "../../api/rs/build" }
 
 [features]
-default = ["slint/default"]
+default = ["slint/default", "weer_api"]
```

Comment:
Maybe we can introduce a feature name for this: 
Something like this.

```suggestion
default = ["slint/default", "network"]
## 
network = ["dep:weer_api"]
```

Reply:
done